components/StyledInput.js

```diff
@@ -37,14 +37,21 @@ const StyledInput = styled.input`
   border-style: ${props => (props.bare ? 'none' : 'solid')};
   box-sizing: border-box;
   outline: none;
+  background-color: ${themeGet('colors.white.transparent.72')};
```

Comment:
What is the goal you're trying to achieve with this line?

Reply:
I was trying to make the custom input have a background color transparent white, not gray